new-client/src/controls/MapSwitcher.js

```diff
@@ -29,6 +29,11 @@ class MapSwitcher extends React.PureComponent {
 
   handleLoading(maps) {
     let { activeMap } = this.appModel.config;
+
+    maps.sort((a, b) =>
```

Comment:
I'm sorting the array in place instead of copying, some people don't like that but in this case I think it's reasonable instead of copying what seem to be rather complex objects.